geth/txqueue/utils.go

```diff
@@ -0,0 +1,28 @@
+package txqueue
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/signal"
+)
+
+var ErrTxQueueRunFailure = errors.New("error running transaction queue")
+
+// HaltOnPanic recovers from panic, logs issue, sends upward notification, and exits
+func HaltOnPanic() {
```

Comment:
So this code is duplicated in `node/utils.go`?

Reply:
Yes. I'm not sure it's even a right thing to have here, but for the sake of keeping the same functionality, I just copied code. Not sure it's worth moving into separate package, I think we'll revisit the queue halting logic soon as well (I don't like that tqQueue send signals here, it's wrong place to handle go-react communication)

Reply:
Makes sense. I just wanted to draw attention so that all of us are on the same page.